Rlog/CoreData/CoreDataManager.swift

```diff
@@ -9,48 +9,199 @@ import CoreData
 
 struct CoreDataManager {
     static let shared = CoreDataManager()
+    private let container = NSPersistentContainer(name: "DataModel")
+    private let cloudContainer = NSPersistentCloudKitContainer(name: "DataModel")
```

Comment:
DataModel 네임을 별도의 property로 가지면 어때요 ?

```suggestion
    private let containerName = "DataModel"
    private lazy var container = NSPersistentContainer(name: containerName)
    private lazy var cloudContainer = NSPersistentCloudKitContainer(name: containerName)
```

Reply:
https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/6/files#r1001672989 과 연관성이 있을듯해서 함께 고려해보시면 좋을듯합니다

Reply:
확인했습니다 

enum 으로 빼서 수정하겠습니다